server/services/utils/getServicesForUser.ts

```diff
@@ -504,6 +510,46 @@ export default (
         userHasRoles([Role.CreateAnEMOrder], roles) &&
         isActiveInEstablishment(activeCaseLoadId, ServiceName.CEMO, activeServices, false),
     },
+    {
+      id: 'allocate-key-workers',
+      heading: 'Key workers',
+      description: 'Allocate key workers to prisoners and manage key work in your establishment.',
+      href: config.serviceUrls.allocateKeyWorkers.url,
+      navEnabled: true,
+      enabled: () =>
+        userHasRoles([Role.OmicAdmin, Role.KeyworkerMonitor], roles) &&
+        isActiveInEstablishment(activeCaseLoadId, ServiceName.ALLOCATE_KEY_WORKERS, activeServices, false),
+    },
+    {
+      id: 'my-key-worker-allocations',
+      heading: 'My key worker allocations',
+      description: 'View your key worker allocations and personal statistics.',
+      href: `${config.serviceUrls.allocateKeyWorkers.url}/staff-profile/${staffId}`,
+      navEnabled: true,
+      enabled: () =>
+        userHasRoles([Role.KeyWorker], roles) &&
+        isActiveInEstablishment(activeCaseLoadId, ServiceName.ALLOCATE_KEY_WORKERS, activeServices, false),
+    },
+    {
+      id: 'allocate-personal-officers',
+      heading: 'Personal officers',
+      description: 'Allocate personal officers to prisoners and manage officer work in your establishment.',
+      href: config.serviceUrls.allocatePersonalOfficers.url,
+      navEnabled: true,
+      enabled: () =>
+        userHasRoles([Role.PersonalOfficerAllocate, Role.PersonalOfficerView], roles) &&
+        isActiveInEstablishment(activeCaseLoadId, ServiceName.ALLOCATE_PERSONAL_OFFICERS, activeServices, false),
+    },
+    {
+      id: 'my-personal-officer-allocations',
+      heading: 'My personal officer allocations',
+      description: 'View your personal officer allocations and personal statistics.',
+      href: `${config.serviceUrls.allocatePersonalOfficers.url}/staff-profile/${staffId}`,
+      navEnabled: true,
+      enabled: () =>
+        userHasRoles([Role.PersonalOfficer], roles) &&
```

Comment:
This should be checking the StaffAllocationPolicies for the KEY_WORKER policy rather than the personal officer view role